File: 3476-FindMinimumOperationsToMakeAllElementsDivisibleByThree/3476-FindMinimumOperationsToMakeAllElementsDivisibleByThree.py
# Last updated: 2/10/2026, 8:47:31 PM
class Solution(object):
    def minimumOperations(self, nums):
        y=0
        see=len(nums)

        for i in range(see):
            if nums[i]%3==0:
                  y+=1
        minus=y-see
        if minus<0:
             return (minus*-1)
        else:
             return (minus)

File: 3476-FindMinimumOperationsToMakeAllElementsDivisibleByThree/3476-FindMinimumOperationsToMakeAllElementsDivisibleByThree.c
// Last updated: 7/7/2026, 10:25:01 PM
int minimumOperations(int* nums, int numsSize) {
    int ops = 0;
    for (int i = 0; i < numsSize; i++) {
        int rem = nums[i] % 3;
        if (rem == 1 || rem == 2) {
            ops += 1;  
        }
    }
    return ops;
}
